docker/ubuntu-20.04-amd64-openssl-1.1.x/Dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG UBUNTU_VERSION="20.04"
+
+FROM ubuntu:${UBUNTU_VERSION}
+
+ARG NODE_VERSION="18.12.1"
```

Comment:
We discussed that this could use the most recent LTS version and that we could have a test for the latest version too